hermes/crates/cardano-chain-follower/src/utils.rs

```diff
@@ -91,3 +93,89 @@ pub(crate) fn blake2b_256(value: &[u8]) -> anyhow::Result<[u8; 32]> {
     b.try_into()
         .map_err(|_| anyhow::anyhow!("Invalid length of blake2b_256, expected 32 got {}", b.len()))
 }
+
+#[allow(dead_code)]
+/// Convert the given value to `blake2b_128` array.
+pub(crate) fn blake2b_128(value: &[u8]) -> anyhow::Result<[u8; 16]> {
+    let h = Params::new().hash_length(16).hash(value);
+    let b = h.as_bytes();
+    b.try_into()
+        .map_err(|_| anyhow::anyhow!("Invalid length of blake2b_128, expected 16 got {}", b.len()))
+}
+
+/// Extracts the CIP-19 bytes from a URI.
+#[allow(clippy::doc_markdown)]
```

Comment:
I would fix this clippy lint.
Probable solution would be make it as this some parts into the "```", like this:
```
 "web+cardano://addr/<cip-19 address string>"
<https://github.com/cardano-foundation/CIPs/tree/6bae5165dde5d803778efa5e93bd408f3317ca03/CPS-0016>
URI = scheme ":" ["//" authority] path ["?" query] ["#" fragment]
```

Reply:
without this clippy lint allowance, I need to fix it to 
`<web+cardano://addr>`
which may mislead how the format of the URI will look like